projects/netgrif-components-core/src/lib/data-fields/time-instance-abstract-field/abstract-time-instance-field.component.ts

```diff
@@ -12,8 +14,24 @@ import {NAE_INFORM_ABOUT_INVALID_DATA} from '../models/invalid-data-policy-token
 export abstract class AbstractTimeInstanceFieldComponent extends AbstractDataFieldComponent {
 
     protected constructor(protected _translate: TranslateService,
+                          protected _adapter: DateAdapter<any>,
```

Comment:
Has to be type DataAdapter<any>? Don't we know the type ahead?

Reply:
its DateAdapter, not DataAdapter, and not sure what time is there, I find this implementation on internet

Reply:
🤷